views/_includes/script.njk

```diff
@@ -26,4 +26,47 @@
 <script>
     if (typeof Promise !== "function" && document.querySelector('details') !== null)
         document.write('<script src="/js/details-element-polyfill.js"><\/script>');
+</script>
+
+<!-- Feedback form -->
+<script>
+    const form = document.getElementById('feedback-form');
+    const confirmation = document.getElementById('feedback-confirmation');
+    const error = document.getElementById('feedback-error');
+    const csrf = document.getElementById('feedback-csrf');
+
+    const submitFeedback = (event) => {
+        event.preventDefault();
+
+        let xmlhttp= window.XMLHttpRequest ?
+            new XMLHttpRequest() : new ActiveXObject("Microsoft.XMLHTTP");
+
+        // handle post-submit (swap form for confirmation)
+        xmlhttp.onreadystatechange = function() {
+            if (this.readyState === 4) {
+                form.classList.add('hidden');
+                if (this.status === 200) {
+                    confirmation.classList.remove('hidden');
+                } else {
+                    error.classList.remove('hidden');
+                }
+            }
+        }
+
+        // gather up the checkbox values
+        var problems = []
+        var checkboxes = form.querySelectorAll('input[type=checkbox]:checked')
+
+        for (var i = 0; i < checkboxes.length; i++) {
+            problems.push(checkboxes[i].value)
+        }
+
+        // post to our feedback endpoint
+        xmlhttp.open('POST', '/{{ getLocale() }}/feedback', true);
```

Comment:
This action path is like this because we need to submit to a language-specific url so we can capture the user's selected language, but the Feedback form renders on the 404 and 500 pages, and the routePath helper is not available on those pages.